Lib/ufo2ft/outlineOTF.py

```diff
@@ -9,6 +11,7 @@
 from fontTools.ttLib.tables.O_S_2f_2 import Panose
 from fontTools.ttLib.tables._h_e_a_d import mac_epoch_diff
 from fontTools.ttLib.tables._n_a_m_e import NameRecord
+from robofab.pens.boundsPen import ControlBoundsPen
```

Comment:
Why don't you use the fonttools' boundsPen here? Let's drop robofab dependency as much as possible..